### ISSUE TITLE
when pasting some items - only the first one is added

### ISSUE THREAD

**Issue by na-le** [Gbuomprisco/ngx-chips#664]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

I added the option to paste items but when pasting some items , only the first one is added, all the others - throw validation error. I think that it is because of the condition 
` this.inputForm.form.valid;`
in 
```
const validationFilter = (tag: TagModel): boolean => {
            const isValid = this.isTagValid(tag, fromAutocomplete) && this.inputForm.form.valid;

            if (!isValid) {
                this.onValidationError.emit(tag);
            }

            return isValid;
        };
```

my definition is:
```
<tag-input class="services-tag-input" [(ngModel)]='specifiedServices.services'
             [animationDuration]="{enter: '0ms', leave: '0ms'}"
             [disable]='isDisabled'
             [editable]="true"
             [addOnPaste]="true"
             [pasteSplitPattern]="','"
             (onAdd)="onAdd($event)"
             (onPaste)="onPaste($event)"
             (onValidationError)="onValidationError($event)"
             [asyncValidators]="validators">

```
trying to paste: a,c,b
the items are valid!! and there are no duplicates!!!

in the console I see:
added vale: a
error in: c
error in: b

**Expected behavior**

all valid elements should be added

**Minimal reproduction of the problem with instructions (if applicable)**

**What do you use to build your app?. Please specify the version**

angular-cli

**Angular version:** 

**ngx-chips version:**

1.3.6 RC.3

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

All

**Comment by na-le** [comment]
I debugged it and saw that, as I thought, `this.inputForm.form.valid` is false
it is false because the status of the form is **PENDING** and not **VALID**
can I do anything with it?

**Comment by yonicozac** [comment]
+1

**Comment by Gbuomprisco** [comment]
Hi, this PR should fix this: https://github.com/Gbuomprisco/ngx-chips/pull/719

**Comment by Gbuomprisco** [comment]
I released v 1.7.6. This was a tough one to tackle :\. Please reopen if you have issues :+1:
